content/2.0.html.haml

```diff
@@ -90,109 +89,121 @@ title: "Jenkins 2.0"
 
 %h2
   Highlights
+%ul 
+  %li Jenkins 2.0 supports delivery pipelines natively. 
+  %li Jenkins 2.0 improves usability.
+  %li Jenkins 2.0 is backwards compatible.
 
-%dl#highlights
-  %dt
-    %strong
-      Pipeline as Code
-    keeps delivery
-    %a{:href => 'https://en.wikipedia.org/wiki/Don%27t_repeat_yourself'}
-      DRY
-  %dd
-    %p
-      The new
-      %a{:href => '/solutions/pipeline'}
-        Pipeline
-      functionality in Jenkins allows you to define configuration as code,
-      which can be checked in and version controlled along with the rest of
-      your project's source code.
-
-    %img{:src => '/images/hello-world-pipeline.png', :title => 'A simple build/test pipeline'}/
-
-    %p
-      Defining your pipeline's configuration as code makes it easier to create
-      a simple "build and test" pipeline, while enabling more advanced and
-      complex pipelines through the expressive Groovy-based domain specific
-      language.
-
-    %p
-      With the new Pipeline Stage View Plugin, the different stages of pipelines
-      can easily be visualized.
-
-    %a{:href => '/images/pipeline-stage-view.png', :target => '_blank'}
-      %img{:src => '/images/pipeline-stage-view.thumb.png', :title => 'A pipeline stage view',
+%h2
+  Pipelines
+
+%h3 Problem
+
+%p As organizations of all types seek to deliver high quality software faster, their use of Jenkins is extending beyond just CI to Continuous Delivery (CD) In order to implement CD, teams  need a flexible way to model and orchestrate their entire delivery pipeline.
+  
+%h3 Solution
+
+%p Jenkins 2.0 now natively supports delivery pipelines as a first-class entity. Pipelines introduces a domain specific language that help Jenkins users to model their software delivery pipelines as code, which can be checked in and version controlled along with the rest of your project's source code.
+
+%img{:src => '/images/hello-world-pipeline.png', :title => 'A simple build/test pipeline'}/
+
+%p An expressive domain specific language (configuration as code) makes it easier and faster to create a simple "build, test and deploy" pipeline or build advanced pipelines. The net result is improved productivity for dev and ops teams.
+
+%h4 Key Features and Benefits for Pipelines
+
+%table{:border => 1} 
+  %tr
+    %th Features
+    %th Benefits
+
+  %tr 
+    %td Pipeline as code (DSL)
+    %td
+      %ul
+        %li Easily define simple and complex pipelines through the DSL in a <i> Jenkinsfile</i>.
+        %li Pipeline as code provides a common language to help teams (e.g. Dev and Ops) work together.
+        %li Easily share pipelines between teams by storing common functions in global libraries
+        
+  %tr
+    %td New Pipeline Stage View
+    %td 
+      %ul
+        %li A new interface addition to make it easy to visualize the progress across an entire pipeline 
```

Comment:
I thought the screenshot of the pipeline stage view was pretty cool, why did you remove it?